ci/ray_ci/macos/macos_ci.sh

```diff
@@ -23,51 +23,48 @@ run_small_and_large_flaky_tests() {
   # shellcheck disable=SC2046
   # 42 is the universal rayci exit code for test failures
   (bazel query 'attr(tags, "client_tests|small_size_python_tests|large_size_python_tests_shard_0|large_size_python_tests_shard_1|large_size_python_tests_shard_2", tests(//python/ray/tests/...))' | select_flaky_tests |
-    xargs bazel test --config=ci $(./ci/run/bazel_export_options) \
-      --test_env=CONDA_EXE --test_env=CONDA_PYTHON_EXE --test_env=CONDA_SHLVL --test_env=CONDA_PREFIX \
-      --test_env=CONDA_DEFAULT_ENV --test_env=CONDA_PROMPT_MODIFIER --test_env=CI) || exit 42
+    xargs ./ci/ray_ci/macos/macos_ci_test.sh) || exit 42
```

Comment:
maybe get the refactoring of this file into another PR? seems that this part (extraction of the `macos_ci_test.sh`) is orthogonal to the bisector logic.

Reply:
sound good